data/members/the-qrl-foundation.yaml

```
@@ -0,0 +1,70 @@
+id: the-qrl-foundation
```

'the' is probably not appropriate here, this is most likely also not the registered foundation name?
```suggestion
id: qrl-foundation
```

---

You will also need to rename the folder and logo as this id needs to be in sync.

---

Foundation is registered in Switzerland with the name of Die QRL Stiftung which in English translates to The QRL Foundation. Die means The in English and Stiftung means the Foundation in English.